src/test/common/utils/interest.ts

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai'
+import { getInterestDetails } from 'common/interest'
+import * as claimStoreServiceMock from '../../http-mocks/claim-store'
+import { Claim } from 'claims/models/claim'
+import { InterestType } from 'claim/form/models/interest'
+import { Moment } from 'moment'
+import { MomentFactory } from 'common/momentFactory'
+
+describe('getInterestDetails', () => {
+
+  it('should return undefined when interest has not been selected', async () => {
+    let claim: Claim = new Claim().deserialize(claimStoreServiceMock.sampleClaimObj)
+    claim.claimData.interest.type = InterestType.NO_INTEREST
+    expect(await getInterestDetails(claim)).to.be.eq(undefined)
+  })
+
+  it('should return expected interestDetails containing the needed values to be displayed', async () => {
```

Comment:
That test scenario is testing too much IMO. Since we changing the date calculation logic I would love to see the following tests:
- 0 days if interest claimed for today's date
- 1 day if interest claimed from yesterday